source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs

```diff
@@ -33,5 +34,22 @@ public override Func<IVariants> Resolve(Document context, string target, Paramet
     public override void RegisterExtensions(IExtensionManager manager)
     {
         manager.Register(nameof(FamilySizeTableManager.GetFamilySizeTableManager), context => FamilySizeTableManager.GetFamilySizeTableManager(context, family.Id));
+        manager.Register(nameof(FamilyUtils.FamilyCanConvertToFaceHostBased), context => FamilyUtils.FamilyCanConvertToFaceHostBased(context, family.Id));
+        manager.Register(nameof(FamilyUtils.GetProfileSymbols), ResolveProfileSymbols);
+    }
+    
+    private static object ResolveProfileSymbols(Document context)
+    {
+        var values = Enum.GetValues(typeof(ProfileFamilyUsage));
+        var capacity = values.Length;
+        var variants = new Variants<ICollection<ElementId>>(capacity);
+        
+        foreach (ProfileFamilyUsage value in values)
+        {
+            variants.Add(FamilyUtils.GetProfileSymbols(context, value, false), value.ToString());
```

Comment:
We have to add extra description to understand context. For example, this method with $"{value} with multiple curve loops" and next $"{value} with single curve loop"

Reply:
Forgot about this PR. I fixed it all